### ISSUE TITLE
[Android] Library is interfering with react-native's Keyboard module

### ISSUE THREAD

**Issue by mstankov** [thebylito/react-native-navigation-bar-color#45]
I've noticed that hiding the navigation bar like this:
`changeNavigationBarColor('transparent', false, true);`
affects the application in such a way that it somehow blocks the Keyboard module's events from firing, e.g. **keyboardDidShow**

Has anyone else faced this issue? I prefer hiding the navigation bar on android devices with soft navigation for better user experience but this is pretty bad because it affects other aspects of the user experience, for example the keyboard hides any inputs it goes over and libraries such as **react-native-keyboard-aware-scroll-view** stops functioning

**Comment by molimat** [comment]
I dont think it is the same problem, but maybe you can solve it usign the same solution.
When I show the keyboard the navigationBar shows up again even I had set hideNavigationBar() on may App.js.
To avoid it, I added an event listener to my componentDidMount (or useEffect(() => {},[]) with the code `   Keyboard.addListener('keyboardDidHide', this._handleAppStateChange);`
